usb.cpp

```diff
@@ -96,12 +96,14 @@ namespace bsp
     void usbDeinit()
     {
         // Restart HW tick for resume operation
-        xTimerStart(usbTick, 1000);
+        xTimerStart(usbTick, 0);
         // Resume if suspended
         composite_resume(usbDeviceComposite);
 
         LOG_INFO("usbDeinit");
         composite_deinit(usbDeviceComposite);
+
+        xTimerStop(usbTick, 0);
```

Comment:
shouldn't the time be stopped before deinit?

Reply:
i think it can be turned off before `composite_deinit` - tickTimer is needed for resume operation

Reply:
@mjan-dev done